doc/index.md

```diff
@@ -588,7 +588,9 @@ provides, and a blurb about how each is intended to be used.
 <DT>\link ranges::view::sliding_fn `view::sliding`\endlink</DT>
   <DD>Given a range and a count `n`, place a window over the first `n` elements of the underlying range. Return the contents of that window as the first element of the adapted range, then slide the window forward one element at a time until hitting the end of the underlying range.</DD>
 <DT>\link ranges::view::split_fn `view::split`\endlink</DT>
-  <DD>Given a source range and a delimiter specifier, split the source range into a range of ranges using the delimiter specifier to find the boundaries. The delimiter specifier can be a value, a subrange, a predicate, or a function. The predicate should take an single argument of the range's reference type and return true if and only if the element is part of a delimiter. The function should accept current/end iterators into the source range and return `make_pair(true, iterator_past_the_delimiter)` if the current position is a boundary; otherwise, `make_pair(false, cur)`. The delimiter character(s) are excluded from the resulting range of ranges.</DD>
+  <DD>Given a source range and a delimiter specifier, split the source range into a range of ranges using the delimiter specifier to find the boundaries. The delimiter specifier can be a value or a subrange. The delimiter character(s) are excluded from the resulting range of ranges.</DD>
```

Comment:
* "a value or a subrange" was slightly confusing *before* we had a library component named `subrange`. "An element or a range of elements" seems better to me.
* "The delimiter character(s)" could more precisely be "The elements matching the delimiter."